src/aleph_client/commands/account.py

```diff
@@ -115,3 +117,37 @@ def export_private_key(
 def path():
     if sdk_settings.PRIVATE_KEY_FILE:
         typer.echo(sdk_settings.PRIVATE_KEY_FILE)
+
+
+@app.command()
+def sign(
+    message: Optional[str] = typer.Option(None, help="Message to sign"),
+    private_key: Optional[str] = typer.Option(
+        sdk_settings.PRIVATE_KEY_STRING, help=help_strings.PRIVATE_KEY
+    ),
+    private_key_file: Optional[Path] = typer.Option(
+        sdk_settings.PRIVATE_KEY_FILE, help=help_strings.PRIVATE_KEY_FILE
+    ),
+    debug: bool = False,
+):
+    """Sign a message using your private key."""
+
+    setup_logging(debug)
+
+    if private_key is not None:
+        private_key_file = None
+    elif private_key_file and not os.path.exists(private_key_file):
+        exit(0)
+
+    if message is None:
+        # take from stdin
+        message = ''.join(sys.stdin.readlines())
```

Comment:
Why do you join lines ?

Reply:
That's the only way to get all the lines from stdin TextIO. Should I instead join them with `\n`?